src/v4l2src/v4l2src.c

```diff
@@ -949,11 +959,19 @@ static void gst_imx_v4l2src_class_init(GstImxV4l2VideoSrcClass *klass)
 
 	g_object_class_install_property(gobject_class, IMX_V4L2SRC_FRAMERATE_NUM,
 			g_param_spec_int("fps-n", "FPS numerator",
-				"Numerator of the framerate at which"
+				"Numerator of the framerate at which "
 				"the input stream is to be captured",
```

Comment:
Sorry for the delay ... I think this patch makes a lot of sense. Just a small improvement: this commit makes the fps-n property pretty much redundant, so perhaps add a note to its GObject property description like "(obsolete; use fps property instead)".